.babelrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
.babelrc define babel configuration for whole project and it's known method. We use .babelrc also in diffrent projects ex. in bulkimport. So IMHO we should use this file for configure babel